src/services/cssHover.ts

```diff
@@ -108,6 +113,52 @@ export class CSSHover {
 			}
 		}
 
+		
+		if (hover) {
+			console.log(this.doesSupportMarkdown());
+			hover.contents = this.convertContents(hover.contents);
+		}
+
 		return hover;
 	}
+
+	private convertContents(contents: MarkupContent | MarkedString | MarkedString[]): MarkupContent | MarkedString | MarkedString[] {
+		if (!this.doesSupportMarkdown()) {
+			if (typeof contents === 'string') {
+				return contents;
+			}
+			// MarkupContent
+			else if ('kind' in contents) {
+				return {
+					kind: 'plaintext',
+					value: contents.value
+				};
+			}
+			// MarkedString[]
+			else if (isArray(contents)) {
+				contents.map(c => {
+					return typeof c === 'string' ? c : c.value;
+				});
+			}
+			// MarkedString
+			else {
+				return contents.value;
+			}
+		}
+
+		return contents;
+	}
+
+	private doesSupportMarkdown() {
+		if (!this.clientCapabilities) {
+			this.supportsMarkdown = true;
```

Comment:
default is true?

Reply:
I assume if client doesn't specify capabilities we should use whatever that's good for us.

Reply:
In JSON-LS, when no capabilities are passed to the LS constructor, we use the LSP 1.0 capabilities (no MarkupContent, but MarkedString).
So it was done backward compatible (I remembered that wrong)